test/e2e/localnet-underlay.go

```diff
@@ -13,41 +13,56 @@ import (
 )
 
 const (
-	bridgeName = "ovsbr1"
-	add        = "add-br"
-	del        = "del-br"
+	defaultOvsBridge = "breth0"
+	secondaryBridge  = "ovsbr1"
+	add              = "add-br"
+	del              = "del-br"
 )
 
-func setupUnderlay(ovsPods []v1.Pod, portName string, nadConfig networkAttachmentConfig) error {
+func configureBridgeMappingsForNode(ovsPod v1.Pod, networkName, bridgeName string) error {
+	return configureBridgeMappings(
+		ovsPod.Name,
+		defaultNetworkBridgeMapping(),
+		bridgeMapping(networkName, bridgeName),
+	)
+}
+
+func setupUnderlayOnSecondaryBridge(ovsPods []v1.Pod, portName string, nadConfig networkAttachmentConfig) error {
 	for _, ovsPod := range ovsPods {
-		if err := addOVSBridge(ovsPod.Name, bridgeName); err != nil {
+
+		if err := addOVSBridge(ovsPod.Name, secondaryBridge); err != nil {
```

Comment:
IMHO, you should receive the name of the bridge as the func's input parameter. Please name it `bridgeName`.